src/core/StickyContainer.tsx

```diff
@@ -27,6 +27,7 @@ export interface StickyContainerProps {
     applyWindowCorrection?: (offsetX: number, offsetY: number, winowCorrection: WindowCorrection) => void;
     renderStickyContainer?: (stickyContent: JSX.Element, index: number, extendedState?: object) => JSX.Element | null;
     style?: StyleProp<ViewStyle>;
+    alwaysStickBottom?: boolean;
```

Comment:
Let's change this to `alwaysStickFooter`?

Reply:
done